lua/apisix/plugins/fault-injection.lua

```diff
@@ -0,0 +1,77 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one or more
+-- contributor license agreements.  See the NOTICE file distributed with
+-- this work for additional information regarding copyright ownership.
+-- The ASF licenses this file to You under the Apache License, Version 2.0
+-- (the "License"); you may not use this file except in compliance with
+-- the License.  You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+local core          = require("apisix.core")
+local sleep         = ngx.sleep
+
+local plugin_name   = "fault-injection"
+
+local schema = {
+    type = "object",
+    properties = {
+        abort = {
+            type = "object",
+            properties = {
+                http_status = {type = "integer", minimum = 200},
+                body = {type = "string", minLength = 0},
+            },
+            required = {"http_status"}
+        },
+        delay = {
+            type = "object",
+            properties = {
+                duration = {
+                    type = "string",
```

Comment:
it should be a `number`

Reply:
If the `delay` object only has one `property`, I think the type of `delay` should be "number".

`delay = {type = "number", maximum = 100, minimum = 0}`